core/common/src/main/java/alluxio/grpc/GrpcServer.java

```diff
@@ -114,4 +115,10 @@ public void awaitTermination() {
   public boolean isServing() {
     return mStarted && !mServer.isShutdown() || !mServer.isTerminated();
   }
+
+  @Override
+  public String toString() {
+    // Netty server impl prints out its bind address.
+    return mServer.toString();
```

Comment:
do we want more info in this string? (like it is a GrpcServer or auth server or anything else?) I don't know if any other info is useful...

Reply:
Makes sense. Added authentication server info as well.